0,ValueError,"setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (4,) + inhomogeneous part.",0,,2023-06-19 12:17:59.972859,"[('display_grouped_icon_array(marbles.group([""surface"", ""size""]), ""marble(s)"")', 2), ('visualizations.display_combinations(grouped_data, individuals_name=individuals_name)', 41)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.11/site-packages/IPython/core/interactiveshell.py"", line 3508, in run_code', '    exec(code_obj, self.user_global_ns, self.user_ns)', '  File ""/tmp/ipykernel_31041/2703615065.py"", line 2, in <module>', '    display_grouped_icon_array(marbles.group([""surface"", ""size""]), ""marble(s)"")', '  File ""/tmp/ipykernel_31041/939419177.py"", line 41, in display_grouped_icon_array', '    visualizations.display_combinations(grouped_data, individuals_name=individuals_name)', '  File ""/home/jovyan/materials-fa22-main/materials/fa22/lab/lab10/visualizations.py"", line 176, in display_combinations', '    displays = make_displayed_rectangles(compute_combination_data(category_counts, individuals_name))', '               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^', '  File ""/home/jovyan/materials-fa22-main/materials/fa22/lab/lab10/visualizations.py"", line 140, in make_displayed_rectangles', '    displays.append_column(""rectangles"", displays.apply(rectangles_for_percentage_range, [""start amount"", ""end amount"", ""color""]))', '                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^', '  File ""/opt/conda/lib/python3.11/site-packages/datascience/tables.py"", line 536, in apply', '    return np.array([fn(*row) for row in rows])', '           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^', 'ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (4,) + inhomogeneous part.']"
1,ValueError,"setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (2,) + inhomogeneous part.",0,,2023-06-19 12:48:45.793303,"[('display_grouped_icon_array(marbles.group(""surface""), ""marble(s)"")', 1), ('visualizations.display_combinations(grouped_data, individuals_name=individuals_name)', 41)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.11/site-packages/IPython/core/interactiveshell.py"", line 3508, in run_code', '    exec(code_obj, self.user_global_ns, self.user_ns)', '  File ""/tmp/ipykernel_31041/2374735592.py"", line 1, in <module>', '    display_grouped_icon_array(marbles.group(""surface""), ""marble(s)"")', '  File ""/tmp/ipykernel_31041/939419177.py"", line 41, in display_grouped_icon_array', '    visualizations.display_combinations(grouped_data, individuals_name=individuals_name)', '  File ""/home/jovyan/materials-fa22-main/materials/fa22/lab/lab10/visualizations.py"", line 176, in display_combinations', '    displays = make_displayed_rectangles(compute_combination_data(category_counts, individuals_name))', '               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^', '  File ""/home/jovyan/materials-fa22-main/materials/fa22/lab/lab10/visualizations.py"", line 140, in make_displayed_rectangles', '    displays.append_column(""rectangles"", displays.apply(rectangles_for_percentage_range, [""start amount"", ""end amount"", ""color""]))', '                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^', '  File ""/opt/conda/lib/python3.11/site-packages/datascience/tables.py"", line 536, in apply', '    return np.array([fn(*row) for row in rows])', '           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^', 'ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (2,) + inhomogeneous part.']"
